TombEngine/Math/Geometry.cpp

```diff
@@ -260,4 +260,9 @@ namespace TEN::Math::Geometry
 
 		return false;
 	}
+
+	const int FastVectorsDistance(int x1, int y1, int z1, int x2, int y2, int z2)
```

Comment:
Why not pass the actual vectors?

Reply:
I also don't see it used anywhere.

Reply:
It was added for saving costly SIMD load register operations. Yes, never used, I could remove it.